test/framework/fileConversion.go

```diff
@@ -37,44 +40,86 @@ func FormatTestData(srcFile, tgtDir string, targetFormats ...string) (string, er
 	return srcFile, nil
 }
 
-func tarCmd(src, tgtDir string) (string, error) {
-	base := filepath.Base(src)
-	tgt := filepath.Join(tgtDir, base+image.ExtTar)
-	args := []string{"-cf", tgt, src}
+func toTar(src, tgtDir string) (string, error) {
+	tgtFile, tgtPath, err := createTargetFile(src, tgtDir, image.ExtTar)
+	defer tgtFile.Close()
 
-	if err := doCmdAndVerifyFile(tgt, "tar", args...); err != nil {
-		return "", err
+	w := tar.NewWriter(tgtFile)
+	defer w.Close()
+
+	srcFile, err := os.Open(src)
+	if err != nil {
+		return "", errors.Wrap(err, "Error opening file")
 	}
-	return tgt, nil
+	defer srcFile.Close()
+
+	srcFileInfo, err := srcFile.Stat()
+	if err != nil {
+		return "", errors.Wrap(err, "Error stating file")
+	}
+
+	hdr, err := tar.FileInfoHeader(srcFileInfo, "")
+	if err != nil {
+		return "", errors.Wrap(err, "Error generating tar file header")
+	}
+
+	err = w.WriteHeader(hdr)
```

Comment:
In other examples I've seen of writing a tar file there's a loop for each file being archived.  Maybe add a comment stating assumption of single file (though I guess that's pretty obvious now that I think about it)...

Reply:
Yeah, the package is designed to handle single files only.  I can add a package level comment to describe the overall design / behavior though.